cnapy/cellnetanalyzer.py

```diff
@@ -157,3 +158,24 @@ def read_config(self):
                 print("Could not find abs_tol in cnapy-config.txt")
         except NoSectionError:
             print("Could not find section cnapy-config in cnapy-config.txt")
+
+    def read_cobrapy_config(self):
+        ''' Try to read data from cobrapy-config.txt into appdata'''
+        config_parser = configparser.RawConfigParser()
+        config_parser.read(self.appdata.cobrapy_conf_path)
+        try:
+            Configuration().solver = config_parser.get('cobrapy-config', 'solver')
+        except:
+            pass
+        try:
+            Configuration().processes = int(config_parser.get('cobrapy-config', 'processes'))
+        except:
+            pass
+        try:
+            val = float(config_parser.get('cobrapy-config', 'tolerance'))
+            if 1e-9 <= val <= 0.1:
+                Configuration().tolerance = val
+            else:
+                raise
```

Comment:
The raise statement is not inside an except clause. A bare raise statement re-raises the last caught exception.
https://docs.python.org/3.8/reference/simple_stmts.html#raise